### DISCUSSION TITLE
Cart total quantity

### DISCUSSION THREAD

**Discussion by frehner** [Shopify/storefront-api-feedback#107] [Ideas & Requests]
It's common to show a cart icon along with number of total items in the cart.  Currently the only way to get this information is to iterate over all the CartLines and get the count for each one, and sum them up. 

It would be nice to add something like `totalCount` to [Cart](https://shopify.dev/api/storefront/2022-04/objects/Cart#fields) to make this easier.

Thanks!

**Comment by corygibbons** [comment]
Looks like this is in [`2022-07`](https://shopify.dev/api/storefront/2022-07/objects/Cart#field-cart-totalquantity) 🎉

**Reply by frehner** [reply]
Oh nice catch, thanks!

**Comment by arobbins** [comment]
Thanks to the Shopify team for adding this. Huge help!
